Sources/__SwiftOnServer_org/Documentation.docc/2024/02/08-using-swiftnio-channels/08-using-swiftnio-channels.md

```diff
@@ -132,7 +117,7 @@ try await withThrowingDiscardingTaskGroup { group in
 This code is an implementation of the server bootstrap that was created in the previous snippet. Let's go over the code step-by-step:
 
 1. Create a task group to manage the lifetime of our server
-2. By calling `executeAndClose`, receive a sequence of incoming clients. Once this sequence ends, the end of the function is reached and the server is closed.
+2. By calling ``NIOAsyncChannel\executeThenClose(_:)``, receive a sequence of incoming clients. Once this sequence ends, the end of the function is reached and the server is closed.
```

Comment:
these expressions won’t parse as codelinks, as `\` is not the path separator, only `/` and `.` are. the codelink format is written down here:

https://swiftinit.org/docs/swift-unidoc/guides/codelinks